DEVELOPER.md

```diff
@@ -3,6 +3,12 @@
 Do this and you should be 👌👌👌:
 
 ```
+bash script/download-libs.sh
 npm ci
 npm test
-```
\ No newline at end of file
+```
+
+
+_notes_ - As a developer you need you download the FFI libraries prior to running `npm install` / `npm ci` as the libraries will be expected to be there, and you won't have any `node_modules` installed yet.
```

Comment:
I think this paragraph would be improved by saying that `script/download-libs.sh` does this download - otherwise someone might think that it was a separate instruction here.

Reply:
shall add, feel free to propose something nice, I just quickly boshed something in whilst it was fresh in my mind, i'll make it more verbose tomorrow and add some decent docs about cirrus-ci, notes about keeping them in sync (gh actions / cirrus ci) and tips for running both locally (via act or cirrus-cli) - depending on the end users machine 👍🏾 

We can massively improve contributions if we improve on the docs on helping people pick up the codebase and just get rocking.

Reply:
I would just change:

> As a developer you need you download the FFI libraries

to

> As a developer, you need to run `script/download-libs.sh` to download the FFI libraries...